src/api/precautionApi.tsx

```diff
@@ -84,6 +84,14 @@ const getUserActivePrecautionEndDate = async (userId: string, type: string) => {
     type,
   });
 };
+
+const getUsersForPrecaution = async () =>{
+  const response = await api.get(`Precaution/getUsersForPrecaution`).catch((error)=>{
```

Comment:
Can't we just return here without assigning new variable?

Reply:
Changed code to just return result.